x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/MonitoringService.java

```diff
@@ -44,7 +45,15 @@
     public static final TimeValue MIN_INTERVAL = TimeValue.timeValueSeconds(1L);
 
     /**
-     * Dynamically controls enabling or disabling the collection of Monitoring data.
+     * Dynamically controls enabling or disabling the collection of Monitoring data only from Elasticsearch.
```

Comment:
Probably worth adding further differentiation in this documentation (for later references):

```java
/*
 * Dynamically controls enabling or disabling the collection of Monitoring data only from Elasticsearch.
 * <p>
 * This should only be used while transitioning to Metricbeat-based data collection for Elasticsearch with
 * {@linkplain #ENABLED} set to {@code true}. By setting this to {@code false} and that value to {@code true},
 * Kibana, Logstash, Beats, and APM Server can all continue to report their stats through this cluster until they
 * are transitioned to being monitored by Metricbeat as well.
 */
```